app/bundles/ConfigBundle/Model/SysinfoModel.php

```diff
@@ -0,0 +1,139 @@
+<?php
+/**
+ * @package     Mautic
+ * @copyright   2014 Mautic Contributors. All rights reserved.
+ * @author      Mautic
+ * @link        http://mautic.org
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace Mautic\ConfigBundle\Model;
+
+use Mautic\CoreBundle\Model\CommonModel;
+
+/**
+ * Class SysinfoModel
+ */
+class SysinfoModel extends CommonModel
+{
+    protected $phpInfo;
+    protected $folders;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getPermissionBase()
+    {
+        return 'config:sysinfo';
```

Comment:
Is this permission defined in a Permissions class? I didn't see one so wanted to double check. If not, we can probably remove this function and just restrict access admins?